seqr/views/apis/users_api_tests.py

```diff
@@ -309,93 +313,88 @@ def assert_has_anvil_calls(self):
         mock.call(self.collaborator_user, fields=WORKSPACE_FIELDS),
     ]
     self.mock_list_workspaces.assert_has_calls(calls)
-    calls = [
-        mock.call(self.no_access_user, 'my-seqr-billing', 'anvil-1kg project n\u00e5me with uni\u00e7\u00f8de'),
-        mock.call(self.no_access_user, 'my-seqr-billing', 'anvil-project 1000 Genomes Demo'),
-    ]
-    self.mock_get_ws_acl.asssert_has_calls(calls)
+    self.mock_get_ws_acl.assert_not_called()
     self.mock_get_ws_access_level.assert_not_called()
 
 
 class AnvilUsersAPITest(AnvilAuthenticationTestCase, UsersAPITest):
     fixtures = ['users', 'social_auth', '1kg_project']
     COLLABORATOR_NAMES = {'test_user_manager', 'test_user_non_staff', 'test_user_pure_anvil@test.com'}
+    LOCAL_COLLABORATOR_NAMES = set()
     NUM_USERS = 4
-    NEW_USER_IDX = NUM_USERS - 2
 
-    def test_get_all_collaborators(self):
-        super(AnvilUsersAPITest, self).test_get_all_collaborators()
+    def test_get_all_collaborator_options(self):
+        super(AnvilUsersAPITest, self).test_get_all_collaborator_options()
         assert_has_anvil_calls(self)
 
-    def test_get_all_staff(self):
-        super(AnvilUsersAPITest, self).test_get_all_staff()
-        self.mock_list_workspaces.asssert_not_called()
-        self.mock_get_ws_acl.asssert_not_called()
+    def test_get_all_staff_options(self):
+        super(AnvilUsersAPITest, self).test_get_all_staff_options()
+        self.mock_list_workspaces.assert_not_called()
+        self.mock_get_ws_acl.assert_not_called()
```

Comment:
Very bad mistakes I made. It is weird that `mock` can't catch the errors.

Reply:
yeah I mean the point of mocks is that you can call any method from them and they don't error so they can be mocking anything, but I wish they had some built in edge cases for common typos, since this is pretty problematic